playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -7,7 +7,7 @@
 # core team tells us not to rely on yum module transaction flattening anyway.
 
 - name: Upgrade master packages
-  package: name={{ master_pkgs | reject('equalto', '') | join(',') }} state=present
+  package: name={{ master_pkgs | join(',') | replace(',,', ',') }} state=present
```

Comment:
This is not a great way of doing this either way.  Let's get rid of the replace bit here.